build-tools/packages/build-cli/src/commands/list.ts

```diff
@@ -103,7 +103,8 @@ export default class ListCommand extends BaseCommand<typeof ListCommand> {
 
 		const filterOptions = parsePackageFilterFlags(this.flags);
 		const packageList = await pnpmList(rgOrPackage.repoPath);
-		const filtered = filterPackages(packageList, filterOptions)
+		const filtered = (await filterPackages(packageList, filterOptions))
+			// eslint-disable-next-line unicorn/no-await-expression-member
```

Comment:
IMO we should disable this rule or reduce it to a warning. But I can also update it to use an intermediate variable if desired.

Reply:
I don't know about the rule. It probably is good for a case like this if you wanted to debug something. Having the local can be useful and doesn't impact the block adversely.

Reply:
Updated in latest.